PODMAN.md

```diff
@@ -2,22 +2,39 @@
 
 Install podman 3.1+, then run it in docker compatible mode:
 
+## Rootless podman
+
 ```
 systemctl start --user podman.socket
 ```
 
-After this command, you can use kubevirtci with the typical `make cluster-*`
-commands.
+Currently rootless podman is **not** working with the `make cluster-sync`
+command, essentially because incoming traffic is coming from the loopback device
+instead of eth0.
+
+The current rules - [0], [1] - allow `make cluster-up` to run successfully, but
```

Comment:
Make [0] and [1] links instead of adding them below?

Reply:
Can I add 2 links to same word ? That's the reason I put this as I did :)

Reply:
Looks good now :)